0_locs_poly_setup/src/get_polygons.R

```diff
@@ -0,0 +1,47 @@
+#' Function to download NHDPlusHR HUC-4 geopackages that contain all NHD layers from 
+#' a HUC
+#' 
+#' @param HUC a 4-digit or longer HUC value
+#' @param minimum_sqkm a numeric threshold value to filter waterbodies, in square 
+#' kilometers; if specified as NA_real_, no filter will be applied
+#' @returns filepath for new {sf} file that contains the filtered polygons
+#' 
+#' 
+#' 
+get_polygons <-  function(HUC, minimum_sqkm) {
+  #check for out folder, create if not present
+  if (dir.exists('0_locs_poly_setup/out/') == FALSE) {
```

Comment:
question: Is the `if()` process necessary here? I'm not sure if the same result would happen without this but it seems possible?

Reply:
because I have the `0_locs_poly_setup/out` folder in .gitignore this is necessary. Alternatively, I could put a `placeholder.txt` in that folder and then gitignore any additional files... Thoughts?

Reply:
Ah! I think I'm getting at something different but lmk if not - does `dir.create()` do essentially the equivalent check and warning on failure, making the `if()` redundant? I'm not clear if you're using it for something more than that here

Reply:
oh, I think I get what you're alluding to now! I think maybe the more simplisitic is to wrap in `try()` like I formerly did with that `tmp` creation that we jettisoned. If you `dir.create()` and it already exists, it doesn't give you a warning, it gives you an error and stops the {targets} run completely. 

This `if()` is overly complicated. I'll send up a commit with this right now.